hyperspy/tests/signal/test_tools.py

```diff
@@ -869,11 +869,8 @@ def test_spikes_removal_tool():
 
     sr = SpikesRemoval(s)
     sr.threshold = 1.5
-    sr.find()
```

Comment:
It seems that the spikes removal tool is now behaving differently, when it shouldn't.